.github/workflows/build-docs.yml

```diff
@@ -10,20 +10,33 @@ on:
     branches: [ '**' ]
 
 jobs:
-  build:
+  pytest:
+    strategy:
+      matrix:
```

Comment:
Do we need a matrix for a single OS version with a single python version?

Reply:
Simplified.